pkgs/applications/networking/cluster/terraform-providers/providers.json

```diff
@@ -972,6 +972,15 @@
     "spdx": "MPL-2.0",
     "vendorHash": "sha256-kh1wv7cuWCC1rP0WBQW95pFg53gZTakqGoMIDMDSmt0="
   },
+  "scalr": {
+    "hash": "sha256-you7ohHHLbTAzwkEiErPSgf8Ppuw+EoZ+CDfOi35J4w=",
+    "homepage": "https://registry.scalr.io/scalr/scalr",
```

Comment:
We currently don't support providers that aren't on registry.terraform.io

Reply:
Oh. How could such support be added?

Reply:
Not very easily. The packaging, scripts/automation assume or hardcode registry.terraform.io in multiple places, also other registries all seem to be just different enough that that they each require special handling.

TBH I'm not enthusiastic about adding support for other registries, it is rather a maintenance burden, especially when they only seem to host a single provider.

cc @zimbatm

Reply:
I'm playing around with the overrideAttrs. I am trying to see if I can add it as a very special provider in the existing framework without having to make a new terraform-provider-scalr package folder.